dirs/dirs.go

```diff
@@ -120,6 +121,7 @@ func SetRootDir(rootdir string) {
 	SnapSocket = filepath.Join(rootdir, "/run/snapd-snap.socket")
 
 	SnapAssertsDBDir = filepath.Join(rootdir, snappyDir, "assertions")
+	SnapAssertsSpoolDir = filepath.Join(rootdir, snappyDir, "spool")
```

Comment:
This should probably be more specific as we may end up with several different spools. Perhaps "auto-import" instead, as it's very specific to that functionality?